Remotion/Web/Development.WebTesting.TestSite.Shared/TreeViewTest.aspx.cs

```diff
@@ -118,13 +118,13 @@ private void InitializeMyTreeViewWithSpecialChildren ()
     }
     private void InitializeTreeViewWithOnlyRootWithSingleQuote ()
     {
-      var root = new TreeNode("With'SingleQuote", "RootValue", "~/Images/SampleIcon.gif");
+      var root = new TreeNode("With'SingleQuote", "RootValue", this.ResolveImageResource("SampleIcon.gif"));
       TreeViewWithOnlyRootWithSingleQuote.Nodes.Add(root);
     }
 
     private void InitializeTreeViewWithOnlyRootWithDoubleQuote ()
     {
-      var root = new TreeNode("With'SingleQuoteAndDouble\"Quote", "~/Images/SampleIcon.gif");
+      var root = new TreeNode("With'SingleQuoteAndDouble\"Quote", "RootValue", this.ResolveImageResource("SampleIcon.gif"));
```

Comment:
"RootValue" here seems to be a copy&paste error?

Reply:
Actually, the code was broken before as image url was passed in as value so I inserted a new value based on the tree view above.